### PR TITLE
Some more renames of syntax nodes

### PR DESCRIPTION
**Pull request by ahoppen**
This is the final round of renames I’m doing for now.

### CODE REVIEW

**Comment by ahoppen** [comment]
@swift-ci Please test

**Review by bnbarham — APPROVED** [review]

File: CodeGeneration/Sources/SyntaxSupport/SyntaxNodeKind.swift [hunk 1]
```diff
@@ -378,7 +378,7 @@ public enum SyntaxNodeKind: String, CaseIterable {
     case .closureCapture: return "closureCaptureItem"
     case .closureCaptureClause: return "closureCaptureSignature"
     case .closureCaptureList: return "closureCaptureItemList"
-    case .closureCaptureSpecifier: return "closureCaptureItemSpecifier"
+    case .closureCaptureModifier: return "closureCaptureItemSpecifier"
```
Comment (bnbarham) [inline CodeGeneration/Sources/SyntaxSupport/SyntaxNodeKind.swift:381]: I don't have a strong opinion on modifier vs specifier here. But seems weird to have both still.

**Comment by ahoppen** [comment]
@swift-ci Please test

**Comment by ahoppen** [comment]
https://github.com/apple/sourcekit-lsp/pull/787

@swift-ci Please test

**Comment by ahoppen** [comment]
https://github.com/apple/sourcekit-lsp/pull/787

@swift-ci Please test Windows

**Comment by ahoppen** [comment]
https://github.com/apple/sourcekit-lsp/pull/787
https://github.com/apple/swift-stress-tester/pull/249

@swift-ci Please test

**Comment by ahoppen** [comment]
https://github.com/apple/sourcekit-lsp/pull/787
https://github.com/apple/swift-stress-tester/pull/249

@swift-ci Please test

**Comment by ahoppen** [comment]
https://github.com/apple/sourcekit-lsp/pull/787
https://github.com/apple/swift-stress-tester/pull/249

@swift-ci Please test macOS

**Review by kimdv — APPROVED** [review]

**Comment by ahoppen** [comment]
@swift-ci Please test macOS

**Comment by ahoppen** [comment]
https://github.com/apple/sourcekit-lsp/pull/787
https://github.com/apple/swift-stress-tester/pull/249

@swift-ci Please test macOS

**Pull request merged**